flask_resources/context.py

```diff
@@ -48,6 +48,7 @@ def __init__(
         self.payload_mimetype = payload_mimetype  # Content-Type
         self.request_args = request_args
         self.request_content = data
+        self.route = {}
```

Comment:
minor: route_args? Is not putting the route per se, but its args no?

Reply:
Hmm I think `request_args` and `route_args` are confusing: both come from the request... And if we want to get away from the request aspect, we would want to remove `request` references too... Something tells me that every variables in the context object is an "argument" anyway (they are all inputs from the context of execution, which for now is a request). :thinking:  I've put a note in https://github.com/inveniosoftware/flask-resources/issues/2 to revisit naming and shelved it for now.

Reply:
Hmm you have a point... yes... this requires more thought.